app/src/main/res/values-ca/strings.xml

```diff
@@ -0,0 +1,47 @@
+<!--
+  ~ Copyright 2019 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+    <string name="menu_filter_by_grow_zone">Filtrar per zona de creixement</string>
+    <string name="plant_detail_image_content_description">Imatge de la planta</string>
+    <string name="my_garden_title">El meu jardí</string>
+    <string name="plant_list_title">Llistat de plantes</string>
```

Comment:
"Llistat" should be "llista" in this case. See https://aplicacions.llengua.gencat.cat/llc/AppJava/index.html?input_cercar=llistat&action=Principal&method=detall_completa&numPagina=1&idHit=10329&database=FITXES_PUB&tipusFont=Fitxes%20de%20l%27Optimot&idFont=10329&titol=llista%20o%20llistat?%20/%20llistat%20de%20convidats%20o%20llista%20de%20convidats?%20/%20llista%20d%27etiquetes%20o%20llistat%20d%27etiquetes?&numeroResultat=3&clickLink=detall&tipusCerca=cerca.tot

```suggestion
    <string name="plant_list_title">Llista de plantes</string>
```